utility/src/main/java/org/oppia/android/util/parser/image/BUILD.bazel

```diff
@@ -0,0 +1,223 @@
+"""
+General-purpose utilities for image loading and processing.
+"""
+
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "bitmap_blurrer",
+    srcs = [
+        "BitmapBlurrer.kt",
+    ],
+    visibility = ["//utility/src/main/java/org/oppia/android/util/parser/svg:__pkg__"],
+    deps = [
+        "//third_party:com_github_bumptech_glide_glide",
+    ],
+)
+
+kt_android_library(
+    name = "bitmap_blur_transformation",
+    srcs = [
+        "BitmapBlurTransformation.kt",
+    ],
+    visibility = ["//utility/src/main/java/org/oppia/android/util/parser/svg:__pkg__"],
+    deps = [
+        ":bitmap_blurrer",
+        "//third_party:com_github_bumptech_glide_glide",
+    ],
+)
+
+kt_android_library(
+    name = "glide_image_loader",
+    srcs = [
+        "GlideImageLoader.kt",
+    ],
+    visibility = ["//:oppia_testing_visibility"],
```

Comment:
added testing visibiltity as target is used by test cases in testing module and app module